src/sql/workbench/services/objectExplorer/browser/dragAndDropController.ts

```diff
@@ -70,18 +78,39 @@ export class ServerTreeDragAndDrop implements IDragAndDrop {
 	 * Called when the drag operation starts.
 	 */
 	public onDragStart(tree: ITree, dragAndDropData: IDragAndDropData, originalEvent: DragMouseEvent): void {
+		let escapedSchema, escapedName, finalString;
 		TreeUpdateUtils.isInDragAndDrop = true;
 		const data = dragAndDropData.getData();
 		const element = data[0];
-		if (element.nodeTypeId === 'Column' || element.nodeTypeId === 'Table') {
-			const escapedSchema = element.metadata.schema?.replace(/]/g, ']]');
-			const escapedName = element.metadata.name?.replace(/]/g, ']]');
-			const finalString = escapedSchema ? `[${escapedSchema}].[${escapedName}]` : `[${escapedName}]`;
+		if (this.checkNodeTypeSingle(element.nodeTypeId)) {
```

Comment:
This is a static function so you shouldn't use this to access it. But also as I suggested above I'd just move these out of the class anyways